python/sglang/srt/layers/quantization/w4afp8.py

```diff
@@ -328,6 +329,41 @@ def apply(
             output *= self.moe_runner_config.routed_scaling_factor
         return StandardCombineInput(hidden_states=output)
 
+    def apply_deepep_ll(
+        self,
+        layer: DeepEPMoE,
+        dispatch_output: DeepEPLLOutput,
+    ) -> torch.Tensor:
+
+        from sglang.srt.layers.moe.cutlass_w4a8_moe import cutlass_w4a8_moe_deepep_ll
+
+        hidden_states, topk_idx, _, masked_m, _ = dispatch_output
```

Comment:
`dispatch_output` was recently refactored. Check this: https://github.com/sgl-project/sglang/pull/11847